### PR TITLE
docs(tutorials): IPv6 on VM's with ESXi hypervisor

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: tutorials/configure-ipv6-virtual-machine-esxi/index.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+tags: compute dedibox esxi vm virtual-machine ubuntu
+categories:
+  - compute
+  - dedibox
+dates:
+  validation: 2022-02-24
+  posted: 2022-02-24
+---
+
+This tutorial explains how to configure IPv6 on virtual machines on ESXi Hypervizors.
```
Comment (RoRoJ) [inline tutorials/configure-ipv6-virtual-machine-esxi/index.mdx:17]:
```suggestion
This tutorial explains how to configure IPv6 on virtual machines on ESXi hypervisors.
```

File: tutorials/configure-ipv6-virtual-machine-esxi/index.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            via: "LOCAL_IPv6"
+            on-link: true
+    ```
+
+8. Apply the new network configuration with the following command:
+    ```
+    netplan apply 
+    ```
+
+9. You now have configured the IP address, but it the IPv6 connectivity is not working yet. Find the gateway for your prefix by running the following command:
```
Comment (RoRoJ) [inline tutorials/configure-ipv6-virtual-machine-esxi/index.mdx:135]:
```suggestion
   You now have configured the IP address, but the IPv6 connectivity is not working yet.

9. Find the gateway for your prefix by running the following command:
```

Comment (RoRoJ) [reply tutorials/configure-ipv6-virtual-machine-esxi/index.mdx:135]: Check formatting of the above doesn't disturb step display

File: tutorials/configure-ipv6-virtual-machine-esxi/index.mdx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    [Install]
+    WantedBy=network.target
+    ```
+5. Enable the service to start automatically after each reboot of the machine. Then start the service manually for the first time.
+    ```
+    systemctl enable dhclient.service
+    systemctl start dhclient.service
+    ```
+
+6. Verify that the service is running by running the following command:
```
Comment (ldecarvalho-doc) [inline tutorials/configure-ipv6-virtual-machine-esxi/index.mdx:88]:
```suggestion
6. Verify that the service is running with the following command:
```

**Review by RoRoJ — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**